pkg/convenience/synchronizer_node/raw_repository.go

```diff
@@ -91,6 +91,138 @@ func NewRawRepository(connectionURL string, db *sqlx.DB) *RawRepository {
 	return &RawRepository{connectionURL, db}
 }
 
+func (s *RawRepository) GetApplicationRef(ctx context.Context, appID uint64) ([]model.ConvenienceApplication, error) {
```

Comment:
What do you think about refactoring this big repository into smaller ones?
(just asking)

Reply:
I was inspired by the solutions proposed by other synchronizers.
What is your proposal?